app/v1/controllers/requestParsers/validators/validations/SelfEmploymentIdValidation.scala

```diff
@@ -16,29 +16,26 @@
 
 package v1.controllers.requestParsers.validators.validations
 
-import v1.models.errors.{MtdError, SelfEmploymentIdFormatError, RuleSelfEmploymentId}
+import v1.models.errors.{ MtdError, SelfEmploymentIdFormatError, RuleSelfEmploymentId }
 
 object SelfEmploymentIdValidation {
 
-  val nonPropertyLossTypes = Seq("self-employment", "self-employment-class4")
+  private val nonPropertyLossTypes = Seq("self-employment", "self-employment-class4")
```

Comment:
Would it be better to have these values set globally somewhere so we aren't just passing strings around in different places?  Global/package object set values would help us maintain them across the endpoints, i.e.
```
object Foo {
   val SELF_EMPLOYMENT = "self-employment"
   val SELF_EMPLOYMENT_CLASS4 = "self-employment-class4"
}

object SelfEmploymentIdValidation {
  import Foo._
  private val nonPropertyLossTypes = Seq(SELF_EMPLOYMENT, SELF_EMPLOYMENT_CLASS4)
 ...
}

```

Reply:
Definitely. (Was hoping to do away with need to do this via use of sealed traits.) Will push what I have to another draft PR for poss discussion Monday...